packages/evo-marko/src/tags/evo-3d-viewer/index.marko

```diff
@@ -0,0 +1,81 @@
+import type { AttrClass, AttrString } from "marko/tags-html";
+import type { ModelViewerElement } from "@google/model-viewer";
+import { load as modelViewerLoad } from "@internal/model-viewer";
+export interface Input extends Partial<ModelViewerElement> {
+    class?: AttrClass;
+    errorText?: AttrString;
+    a11yLoadText?: AttrString;
```

Comment:
Now that we don't do any automatic case conversion, it's safe to make attributes mandatory. Should we be enforcing `a11yLoadText` and `errorText` with TypeScript?